pages.id/sunos/svcadm.md

```diff
@@ -0,0 +1,24 @@
+# svcadm
+
+> Instansi untuk manipulasi servis.
```

Comment:
```suggestion
> Instansi untuk manipulasi hak akses layanan.
```

1. Change `servis` to `layanan` for subsequent occurrences.
2. `permission` here is very close to the definition of `hak akses` (`access rights`), so adding additional context like this could help.